File: fixtures/code_samples/csharp/Sample.cs
namespace Example;

public class Config
{
    public string Name { get; set; }
    public int Value { get; set; }

    public Config(string name, int value)
    {
        Name = name;
        Value = value;
    }

    public string Display()
    {
        return $"{Name}: {Value}";
    }
}

public interface IProcessor
{
    string Process(string input);
    bool Validate();
}

public enum Status
{
    Active,
    Inactive,
    Pending
}

File: fixtures/code_samples/php/sample.php
<?php

class Config {
    public string $name;
    public int $value;

    public function __construct(string $name, int $value) {
        $this->name = $name;
        $this->value = $value;
    }

    public function display(): string {
        return "{$this->name}: {$this->value}";
    }
}

interface Processor {
    public function process(string $input): string;
    public function validate(): bool;
}

function helper(int $x): int {
    return $x * 2;
}

$cfg = new Config("test", 42);
echo $cfg->display();
echo helper(10);
